root/static/scripts/common/components/FilterForm.js

```diff
@@ -92,6 +101,19 @@ const FilterForm = ({form}: Props): React.Element<'div'> => {
               </tr>
             ) : null}
 
+            {showACDroppedMessage && showArtistCreditForm ? (
+              <tr>
+                <td />
+                <td>
+                  {exp.l(
+                    `<strong>Warning:</strong> The artist credit filter
+                     was dropped because the list of available artist credits
+                     changed.`,
```

Comment:
The list will always change *except* when moving between all and official if the artist has only official stuff :) It seemed like it'd be true enough to say it changes, but I dunno.

Reply:
As you seem to confirm that it is not necessarily true, and since nothing actually changed in the database, I would suggest to replace “changed” with “may not be the same”.

Reply:
It looks more clear to me, thanks.